ssh/README.ssh

```diff
@@ -0,0 +1,6 @@
+Add your ssh authorized keys files you want to bake
```

Comment:
not installed by CMakeList.txt?

Reply:
where would that go? does this feature even work?